{"uid":"9a02ce577db59ca9","name":"test_edit_contact_empty_data","fullName":"test.Contactos.test_edit_contact#test_edit_contact_empty_data","historyId":"e218ad908714a2195329772962ced733","time":{"start":1726781441914,"stop":1726781441914,"duration":0},"status":"skipped","statusMessage":"XFAIL This test case is expected to fail due to known issue.\n\nTypeError: PayloadContact.build_payload_add_contact() missing 3 required positional arguments: 'firstName', 'lastName', and 'name'","statusTrace":"setup_edit_contact = ({'Espo-Authorization': 'YWRtaW46ODAwNDkwNTM=', 'X-Api-Key': '4c29fe9482f3bccb7ac0a7d04015a2e7'}, {'accountIsInactive': None, 'accountName': None, 'accountsColumns': {}, 'accountsIds': [], ...})\n\n    @pytest.mark.regression\n    @pytest.mark.functional\n    @pytest.mark.xfail(reason=\"This test case is expected to fail due to known issue.\", condition=True)\n    def test_edit_contact_empty_data(setup_edit_contact):\n        headers, contact = setup_edit_contact\n>       payload = PayloadContact().build_payload_add_contact(salutationName=\"Ms.\",\n                                                             addressPostalCode=\"0000\", addressStreet=\"none\",\n                                                             addressState=\"none\", addressCity=\"none\", addressCountry=\"none\",)\nE       TypeError: PayloadContact.build_payload_add_contact() missing 3 required positional arguments: 'firstName', 'lastName', and 'name'\n\ntest/Contactos/test_edit_contact.py:61: TypeError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"get_headers","time":{"start":1726781434606,"stop":1726781434606,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"setup_edit_contact","time":{"start":1726781441717,"stop":1726781441913,"duration":196},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"afterStages":[{"name":"setup_edit_contact::0","time":{"start":1726781441918,"stop":1726781442096,"duration":178},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"tag","value":"@pytest.mark.xfail(reason='This test case is expected to fail due to known issue.', condition=True)"},{"name":"tag","value":"functional"},{"name":"tag","value":"regression"},{"name":"parentSuite","value":"test.Contactos"},{"name":"suite","value":"test_edit_contact"},{"name":"host","value":"fv-az2036-179"},{"name":"thread","value":"2098-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test.Contactos.test_edit_contact"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["functional","regression","@pytest.mark.xfail(reason='This test case is expected to fail due to known issue.', condition=True)"]},"source":"9a02ce577db59ca9.json","parameterValues":[]}